WordPress/Classes/Services/AccountService.m

```diff
@@ -256,6 +256,10 @@ - (void)updateUserDetailsForAccount:(WPAccount *)account success:(void (^)())suc
 
 - (id<AccountServiceRemote>)remoteForAccount:(WPAccount *)account
 {
+    if (account.restApi == nil) {
```

Comment:
I think before https://github.com/wordpress-mobile/WordPress-iOS/commit/977436bfdf8785ac47bcf001a0f46ffce30558fb `restApi` would never be `nil`, so it made no sense to check. 

I ❤️ nullability and optionals so much 😄